### DISCUSSION TITLE
Issue with the default key installing superset on Mac.

### DISCUSSION THREAD

**Discussion by AlejandroMontanosourcetoad** [apache/superset#28375] [Q&A / Help]
I'm using PyCharm's terminal to install Superset. Before generating the user, it asks me to change the default key in the superset_config.py file. However, when I enter the new key, it doesn't seem to capture it and keeps showing me the same message. Can anyone help me?"

**Comment by goldjee** [comment]
Create superset_config.py in your Superset folder and add a line to it, as follows:

```Python
SECRET_KEY = 'your_secret_goes_here'
```

In terminal, right before starting Superset app, invoke this command (assuming you've already cd'd to Superset folder):

```Bash
export SUPERSET_CONFIG_PATH=./superset_config.py
```
